Seed files for broken link checker job

leleamol opened the pull request:
These are the seed files to for broken link checker nightly job.
These files will be referenced by the nightly job to find regressions. @marcoabreu

marcoabreu commented:
Could you elaborate what the output file is for? Please also add a README to the new folder.

leleamol commented:
@marcoabreu  The blc_output.txt file was not needed. I have removed it.
The url_list.txt contains the list of URLs that are publicly accessible.
I have added the README.md file as well.

marcoabreu commented:
Thank you. 

Please work together with Aaron to get this merged. His script automatically purges all content of this repository, so he will have to add an exception for this directory.

aaronmarkham commented:
@leleamol Is this still needed?
